src/__mocks__/index.tsx

```diff
@@ -77,6 +81,7 @@ export const useScreencast = () => {
     simulcastConfig: NOOP,
     setScreencastTrackEncodingBandwidth: NOOP,
     setScreencastTrackBandwidth: NOOP,
+    setVideoTrackBandwidth: NOOP,
```

Comment:
wait, it's not in useScreeencast 🤔

Reply:
yeah, good catch, I always mix up simulcast and screencast